ib_manifest_util/util.py

```diff
@@ -136,3 +142,83 @@ def download_files(urls: str | list) -> list:
         logger.info(f"Downloaded file {filename} from {address}.")
 
     return filenames
+
+
+def verify_local_channel_environments(
+    local_channel_env: str | Path,
+    offline_mode: bool = True,
+    conda_binary_loc: str | Path | None = None,
```

Comment:
This is very clever and insightful of you! We'll need this to properly test against specific conda/micromamba versions.